nrich-registry/src/test/java/net/croz/nrich/registry/data/controller/RegistryDataControllerTest.java

```diff
@@ -108,6 +108,21 @@ void shouldCreateRegistryEntity() throws Exception {
             .andExpect(jsonPath("$.name").value(entityName));
     }
 
+    @Test
+    void shouldCreateRegistryEntityWithAssociation() throws Exception {
```

Comment:
Maybe `shouldCreateRegistryEntityAndReturnWithInitializedAssociation` or just `shouldCreateRegistryEntityWithInitializedAssociation` (if this is clear enough).